todo-app/src/App.js

```diff
@@ -1,13 +1,51 @@
-import React, { Component } from 'react';
+// import React, { Component } from 'react';
 
-class App extends Component {
-  render() {
+// class App extends Component {
+//   render() {
+//     return (
+//       <div>
+//         Hello World!
+//       </div>
+//     );
+//   }
+// }
+
+// export default App;
+
+
+import React from 'react'
+import './App.css'
+import Todo from './todo.js'
+import './todo.js'
+
+export default class App extends React.Component {
+  render(){
     return (
       <div>
-        Hello World!
+        <h1>Todo:</h1>
+        <Todo
+          username='Ali'
+          imageType='retro'
+          task='Get out of the bed'
+          TaskDate={new Date(2017, 9, 12).toDateString()}
```

Comment:
By convention props, should be [camel cased](https://www.quora.com/What-is-the-difference-between-Pascal-Case-and-Camel-Case) for example `taskDate`!